src/main/java/com/rookout/tutorial/TodoController.java

```diff
@@ -33,7 +33,7 @@
         newTodoRecord.setId(UUID.randomUUID().toString());
         logger.info("Adding a new todo: {}", newTodoRecord);
         // The bug in here in is for the bughunt example
-        String todoTitle = newTodoRecord.getTitle().replaceAll("[^a-zA-Z0-9]+", "");
+        String todoTitle = newTodoRecord.getTitle().replaceAll("[^a-zA-Z0-9 .,!<>]+", "");
```

Comment:
use `\s` for whitespace and not just ' ' :)

Reply:
done